client/my-sites/plans-features-main/index.jsx

```diff
@@ -536,29 +417,66 @@ export class PlansFeaturesMain extends Component {
 		);
 	}
 
-	render() {
-		const { displayJetpackPlans, isInSignup, siteId, plansWithScroll } = this.props;
-		let faqs = null;
+	mayRenderFAQ() {
+		const { displayJetpackPlans, isInSignup } = this.props;
 
-		if ( ! isInSignup ) {
-			faqs = displayJetpackPlans ? <JetpackFAQ /> : <WpcomFAQ />;
+		if ( isInSignup ) {
+			return null;
 		}
 
+		return displayJetpackPlans ? <JetpackFAQ /> : <WpcomFAQ />;
+	}
+
+	render() {
+		const {
+			hideFreePlan,
+			isInSignup,
+			siteId,
+			siteSlug,
+			plansWithScroll,
+			isEligibleForMonthlyPricing,
+		} = this.props;
+		const shouldHideFreePlanBanner = hideFreePlan || ! plansWithScroll;
+		const basePlansPath = isInSignup ? window.location?.pathname : `/plans/${ siteSlug }`;
+
 		return (
 			<div className="plans-features-main">
-				<HappychatConnection />
-				<div className="plans-features-main__notice" />
-				{ ! plansWithScroll && this.renderToggle() }
-				{ plansWithScroll && this.renderFreePlanBanner() }
 				<QueryPlans />
 				<QuerySites siteId={ siteId } />
 				<QuerySitePlans siteId={ siteId } />
-				{ this.getPlanFeatures() }
-				{ this.renderProductsSelector() }
+				<HappychatConnection />
+				<div className="plans-features-main__notice" />
+
+				<ExperimentActivator
+					name="monthly_pricing_test_phase_1"
+					isActive={ isEligibleForMonthlyPricing }
+				>
+					<DefaultVariation name="control">
+						<PlanTypeSelector
+							{ ...this.props }
+							basePlansPath={ basePlansPath }
+							isMonthlyPricingTest={ false }
+						/>
+						<FreePlanBanner { ...this.props } hidden={ shouldHideFreePlanBanner } />
+						{ this.getPlanFeatures( { isMonthlyPricingTest: false } ) }
+						{ this.renderProductsSelector() }
+					</DefaultVariation>
+					<Variation name="treatment">
+						<PlanTypeSelector
+							{ ...this.props }
+							basePlansPath={ basePlansPath }
+							isMonthlyPricingTest={ true }
+						/>
+						{ this.getPlanFeatures( { isMonthlyPricingTest: true } ) }
+						{ this.renderProductsSelector() }
+						<FreePlanBanner { ...this.props } hidden={ shouldHideFreePlanBanner } />
+					</Variation>
+				</ExperimentActivator>
```

Comment:
Hi @taggon, I think something like this is more suitable:

```suggestion
				<Experiment
					name="monthly_pricing_test_phase_1"
				>
					<DefaultVariation name="control">
						<PlanTypeSelector
							{ ...this.props }
							basePlansPath={ basePlansPath }
							isMonthlyPricingTest={ false }
						/>
						<FreePlanBanner { ...this.props } hidden={ shouldHideFreePlanBanner } />
						{ this.getPlanFeatures( { isMonthlyPricingTest: false } ) }
						{ this.renderProductsSelector() }
					</DefaultVariation>
					<Variation name="treatment">
						<PlanTypeSelector
							{ ...this.props }
							basePlansPath={ basePlansPath }
							isMonthlyPricingTest={ isEligibleForMonthlyPricing }
						/>
						{ this.getPlanFeatures( { isMonthlyPricingTest: isEligibleForMonthlyPricing } ) }
						{ this.renderProductsSelector() }
						<FreePlanBanner { ...this.props } hidden={ shouldHideFreePlanBanner } />
					</Variation>
				</Experiment>
```

Here I have:

1. Changed from `ExperimentActivator` to `Experiment`. 

    It seems like possibly `ExperimentActivator` exists to avoid loading state, has that been an issue for you?

    It also might exist to avoid unnecessary assignment? An experiment assignment in calypso is cheap and since you will be using an "exposure event" to track when a user has been exposed to an experiment rather than assignment, when an assignment happens won't matter. If this is the case feel free to use the `Experiment` component as many times as you would like, it might help to think of it as just checking the assignment rather than causing an assignment.

2. Moved the eligibility logic within the `treatment` variation.

    This should help simplify things as the experimentation system shouldn't need to know about this, only the treatment case. It might even help to pass this further below in two spots rather than here, but I don't know this code well enough to say for sure.